scripts/ci/ci_test.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+if ! test -x "$1/labwc"; then
+	echo "$1/labwc not found"
+	exit 1
+fi
+
+args=(
+	"$1/labwc"
+	-C scripts/ci
+	-d
+)
+
+export XDG_RUNTIME_DIR=$(mktemp -d)
+export WLR_BACKENDS=headless
+
+echo "Starting ${args[@]}"
+"${args[@]}"
```

Comment:
Capturing output will let us exit without any noise if everything is happy:
```suggestion
output="$("${args[@]}" 2>&1)"
ret=$?

if test $ret -eq 0; then
    echo "labwc terminated cleanly"
    exit 0
fi

echo "${output}"
```

Reply:
True but I think that having the debug log in there doesn't really harm, as long as the job didn't fail I doubt anyone is actually looking at the logs in the first place.